src/services/auth.ts

```diff
@@ -3,40 +3,34 @@ import PublicClientApplication, { MSALAccount, MSALResult, MSALSilentParams } fr
 import type {
   MSALConfiguration,
 } from "react-native-msal";
-import { Platform } from "react-native";
-
-//import { Platform } from "react-native";
 
 export let pca: PublicClientApplication | null = null;
 
-export async function msalInit(clientId: string, redirectUri: string, authority?: string) {
-
+export async function msalInit(authority: string, clientId: string, redirectUri: string) {
   const config: MSALConfiguration = {
     auth: {
+        authority: authority,
         clientId: clientId,
-        authority: authority ? authority : "https://login.microsoftonline.com/statoilsrm.onmicrosoft.com/",
-        redirectUri: Platform.OS === 'web' ? 'http://localhost:19006' : `msauth.${redirectUri}://auth`,
-        // redirectUri: 'http://localhost:19006'
-        // knownAuthorities: ["https://login.microsoftonline.com/statoilsrm.onmicrosoft.com/"]
+        redirectUri: redirectUri,
     },
     cache: { cacheLocation: 'localStorage' },
   };
   pca = new PublicClientApplication(config);
   await pca.init().catch(e => console.warn(e));
 }
 
-export function msalIsConnected(): boolean {
+export function isMsalConnected(): boolean {
     return !!pca;
 }
 
 
-export async function msalLogin(scope: string) {
+export async function msalLogin(scopes: string[]) {
   if (!pca) {
     throw new Error(
       "Unable to authenticate, pca is null"
     );
   }
-  const result: MSALResult | undefined = await pca.acquireToken({ scopes: [scope] });
+  const result: MSALResult | undefined = await pca.acquireToken({ scopes: scopes });
```

Comment:
`{ scopes: scopes }` can be simplified to `{ scopes }`. Not important though

Reply:
Thanks! Fixed this💪